pwa/src/pages/objects/[id].tsx

```diff
@@ -19,7 +19,7 @@ const IndexPage = (props) => {
   }, [context]);
 
   return (
```

Comment:
Je hebt dit bestand hernoemd waardoor de `route` in de navigatie (`/entities`) breekt. Waardoor de navigatie nu niet meer werkt.